### ISSUE TITLE
Fix Javadoc errors

### ISSUE THREAD

**Issue by macalinao** [macalinao/cloudgame#9]
Let's have better documentation, since there's developers who are going to use this. Fix all of the errors that appear when we compile.

**Comment by macalinao** [comment]
yeah before we release this to developers, we need some decent javadocs.

**Comment by ghost** [comment]
could you please generate some simple javadocs or write a getting startet? that would be really helpfully
